recipes/boost/all/conanfile.py

```diff
@@ -2034,6 +2035,14 @@ def filter_transform_module_libraries(names):
 
                 self.cpp_info.components[f"numpy{pyversion.major}{pyversion.minor}"].requires = ["numpy"]
 
+            if not self.options.get_safe("without_process") and self.settings.os == "Windows":
+                self.cpp_info.components["process"].system_libs.extend(["ntdll", "shell32", "Advapi32", "user32"])
+                if self._shared:
```

Comment:
This is probably already handled by recipe through "BOOST_ALL_DYN_LINK"

Reply:
That is what I would expect as well, but then I ran into errors complaining about missing symbols on Windows (see [here](https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/25388/3-windows-visual_studio/boost/1.86.0//summary.json)) and noticed that in boost::process CMakeLists.txt linking to those libraries is just wrapped around an `if(WIN32)` check - [link](https://github.com/boostorg/process/blob/e8b59f66e6243aa84f28e984705fed5b120f6524/CMakeLists.txt#L59-L61).

So I believe explicitly link to those libs in the `conanfile.py` when building boost::process is not optional/redundant.